master/buildbot/data/builds.py

```diff
@@ -166,6 +150,20 @@ class BuildsEndpoint(Db2DataMixin, base.BuildNestingMixin, base.Endpoint):
     """
     rootLinkName = 'builds'
 
+    fieldMapping = {
```

Comment:
This does not need to be a member. Make it global variable. See how master/buildbot/data/codebases.py does it.